### PR TITLE
Fix batch-norm after reshape

### PR DESCRIPTION
**Pull request by mayokaze**
This pr fixed covert BatchNorm layer after reshape(and convs) by adding transposes before and after. However I think it's just a workaround. The root cause might be in the _convert_reshape dim mapping, but I don't know how to fix it. Could u please have a look? @aseemw

### CODE REVIEW

**Comment by aseemw** [comment]
It is fine as a workaround
